rules/xcodeproj.bzl

```diff
@@ -115,18 +118,28 @@ def _xcodeproj_aspect_impl(target, ctx):
     else:
         srcs = []
         asset_srcs = []
+        other_cflags = []
         for attr in _dir(ctx.rule.files):
             if attr == "srcs":
                 srcs += getattr(ctx.rule.files, attr, [])
             else:
                 asset_srcs += getattr(ctx.rule.files, attr, [])
         srcs = [f for f in srcs if _is_current_project_file(f)]
         asset_srcs = [f for f in asset_srcs if _is_current_project_file(f)]
+        if CcInfo in target:
+          for fi in target[CcInfo].compilation_context.framework_includes.to_list():
+            if fi[0] != '/':
+              fi = "$BAZEL_WORKSPACE_ROOT/%s" % fi
+            other_cflags.append("-F%s" % fi)
+          other_cflags.extend(["-D%s" % di for di in target[CcInfo].compilation_context.defines.to_list()])
+          other_cflags.append("-fobjc-arc") # TODO: only apply to arc sources
```

Comment:
I know you have a TODO here, but we'll likely want to mimic what cocoapods does and default to `CLANG_ENABLE_OBJC_ARC=YES`,  and then set `-fno-objc-arc` on the individual no-arc srcs